embassy-rp/src/gpio.rs

```diff
@@ -956,6 +991,44 @@ impl_pin!(PIN_27, Bank::Bank0, 27);
 impl_pin!(PIN_28, Bank::Bank0, 28);
 impl_pin!(PIN_29, Bank::Bank0, 29);
 
+#[cfg(feature = "rp235xb")]
```

Comment:
this needs adjusting (probably renaming...) `BANK0_PIN_COUNT` / `BANK0_WAKERS`, or async `wait_for_*` will panic due to OOB.

Reply:
I've done the cfg bits, but didn't rename anything. Pi still calls all 48 pins on the B bank 0. I think it's because of the gpio co-processor that lets you access all 48 pins in a single cycle. On the other hand, that's not implemented here yet...

Reply:
> Pi still calls all 48 pins on the B bank 0.

Huh, okay.